omero/developers/testing.txt

```diff
@@ -362,18 +362,27 @@ To view all available markers the :option:`--markers` option can be used with
 :program:`setup.py test` or :program:`py.test` as detailed in
 :ref:`running-python-tests-directly`.
 
-There are currently two custom markers defined:
+There are a small number of custom markers defined:
 
-  `long_running`: for long running tests. These are tests that typically take
-  10 minutes or more and are normally excluded from the daily integration
-  builds.
+.. glossary::
 
-  `broken`: for broken tests. These are tests that are either broken or fail
-  intermittently with no obvious quick fix. Broken tests are excluded from the
-  main integration builds and instead are run in a separate daily build.
-  `broken` markers should have a reason, an associated ticket number or both.
-  If there are multiple associated tickets then a comma-separated list should
-  be used.
+    `long_running`
+        Used to mark tests as long running. These are tests that typically take
+        10 minutes or more and are normally excluded from the daily integration
+        builds.
+
+    `broken`
+        Used to mark broken tests. These are tests that fail consistently with no
+        obvious quick fix. Broken tests are excluded from the main integration builds
+        and instead are run in a separate daily build. `broken` markers should have a
+        reason, an associated ticket number or both. If there are multiple associated
+        tickets then a comma-separated list should be used.
+
+    `intermittent`
+        Used to merk tests that fail intermittently. `intermittent` markers should
+        have a reason, an associated ticket number or both. If there are multiple
+        associated tickets then a comma-separated list should be used. Tests marked
+        as intermittent are normally included in the daily integration builds.
```

Comment:
Yes, I wondered about the wording here. Job config has the other two excluded. I wondered about dropping the 'normally' and just updating the docs if we ever separate out `intermittent` tests. My intention is to wrap these intermittent tests with the flaky decorator so they should always be in the main test build. I'll reword.